crates/rbuilder/Cargo.toml

```diff
@@ -71,9 +71,10 @@ secp256k1 = { version = "0.27.0", features = [
 ] }
 rayon = "1.8.0"
 flate2 = "1.0.27"
-mev-share-sse = "0.1.5"
+mev-share-sse = { git = "https://github.com/paradigmxyz/mev-share-rs", rev = "9eb2b0138ab3202b9eb3af4b19c7b3bf40b0faa8", default-features = false }
```

Comment:
Suggestion: add a PR comment explaining why any crate dependency updates were necessary.

Reply:
agree, so we know when to remove it

Reply:
Good point, so when I tried to use `https://github.com/ralexstokes/ethereum-consensus` for the api It needed to lock this specific version.

Reply:
Can we add the comment on the toml just above this line?